src/adapter/src/catalog/storage.rs

```diff
@@ -1648,6 +1707,20 @@ struct TimestampValue {
 }
 impl_codec!(TimestampValue);
 
+#[derive(Clone, Message, PartialOrd, PartialEq, Eq, Ord, Hash)]
+struct ServerConfigurationKey {
+    #[prost(string)]
+    name: String,
+}
+impl_codec!(ServerConfigurationKey);
+
+#[derive(Clone, Message, PartialOrd, PartialEq, Eq, Ord)]
+struct ServerConfigurationValue {
+    #[prost(string)]
+    value: String,
```

Comment:
Could this be a `Vec<u8>`?

Reply:
Yeah, I originally had it as a `Vec<u8>`, but then changed it to String so it would be consistent with `mz_system_configuration`. Though I think since we're probably going to change `mz_system_configuration`, it makes sense to put this back as a `Vec<u8>`.

Reply:
Just switched it.